src/test/java/uk/gov/di/ipv/cri/common/library/service/AuditServiceTest.java

```diff
@@ -28,6 +30,8 @@
 class AuditServiceTest {
     public String SQS_QUEUE_URL = "https://example-queue-url";
 
+    public String SQS_PREFIX = "TEST";
```

Comment:
mark SQS_PREFIX and SQS_QUEUE_URL as private if they are not used outside this class ( which they shouldn't be )